apps/cowswap-frontend/src/modules/yield/state/poolsInfoAtom.ts

```diff
@@ -19,30 +19,26 @@ type PoolInfoState = {
 
 export type PoolInfoStates = Record<string, PoolInfoState>
 
-type PoolInfoStatesPerAccount = Record<string, PoolInfoStates>
-
-type PoolsInfoState = Record<SupportedChainId, PoolInfoStatesPerAccount | undefined>
+type PoolsInfoState = Record<SupportedChainId, PoolInfoStates | undefined>
 
 const poolsInfoAtom = atomWithStorage<PoolsInfoState>(
-  'poolsInfoAtom:v0',
+  'poolsInfoAtom:v1',
```

Comment:
Still unreleased. I would not bump it.

Reply:
I will reset them before merging in develop.
I usually up them while breaking changes because Elena do tests and it may causes some problems on her side.